.gitmodules

```diff
@@ -260,3 +260,4 @@
 [submodule "plugins/SanguineMutants"]
 	path = plugins/SanguineMutants
 	url = https://github.com/Bloodbat/SanguineMutants.git
+	branch = Cardinal
```

Comment:
Why the separate branch?

Reply:
To keep me sane. Every feature will be made available to Cardinal; but my mind works better if I keep things organized like that. I hope that makes sense.

Reply:
I think it makes sense for development purposes, but if your main branch is compatible to both Rack and Cardinal then in my opinion we should use the main branch in Cardinal by default. For us this makes it much clearer what state the module is in, as we then basically track your official releases.

You can always switch to your Cardinal specific work when you are working on it.

It's just a bit of an odd and uncommon way of organizing this, so for me it doesn't really make sense no ;)

Reply:
Just to be clear: if I don't use my main branch, Sanguine Mutants won't be accepted into Cardinal?

Reply:
@Bloodbat no of course not, but it would be nice to see some good motivations for your approach. And for adding the modules to Cardinal as well (we already have some MI variants, why should we need more? what does this collection bring to the pile etc.)

Reply:
For starters: polyphonic Rings clone; along with others, such as the 3 warps variants; and the 2 Braids variants, with more on the way. End users seem to like that.

Plaits with the latest firmware and the new synth. Models, along with the ability to load custom data.

It fixes  omissions and errors from the Audible Instruments ones. It offers alternative firmwares, some not available in other plugins.

I want Cardinal users to be able to enjoy the collection as much as I do.

As for the branch issue: if it matters that much to you (isn't git supposed to be automatic and pull from the configured branch rendering this conversation moot?) I'll switch it back, though, if I understood the process correctly, when I release a new version, the burden's on me to update Cardinal with it: make a new branch, make a pull request and... ping you (how?)... some of the documentation I found a bit opaque; but that's not relevant to the matter at hand. If I'm wrong and you do the branch tracking and update Cardinal when I release, I'll switch it back, happily.

Reply:
@Bloodbat no worries, just found it odd that we can't use your main branch when it is identical :)